src/main/java/ch/wisv/areafiftylan/security/authentication/AuthenticationController.java

```diff
@@ -82,14 +82,6 @@ public AuthenticationController(UserService userService, AuthenticationService a
         return createResponseEntity(HttpStatus.OK, "Token is valid!");
     }
 
-    @PreAuthorize("isAuthenticated()")
-    @PostMapping("/logout")
```

Comment:
I'm not sure I understand anymore now.

We remove the logout postmapping. How does a user logout now?

Reply:
It's mapped to a native Spring Filter (as it's always been, this endpoint was never reachable) . See the security configuration

Reply:
Ah I see. It makes sense to me now 👍